### ISSUE TITLE
Click event triggered by `.click()` has `isTrusted` prop evaluate to `false`, so click does not perform in v-select component.

### ISSUE THREAD

**Issue by Mikilll94** [cypress-io/cypress#8376]
### Current behavior:
I am using Vue + Vuetify. I have a problem with `v-select` component. If I click on `v-select` the list opens. However, Cypress does not close it when clicking outside of this component (in the repro project, Cypress clicks on Vuetify logo but for other elements this is also not working)

![image](https://user-images.githubusercontent.com/17653648/90906039-5875b300-e3d1-11ea-97bf-e60e307b865c.png)

It does not work even if I use `cy.wait(300)` after opening the list (you can see this in the repro project)

### Desired behavior:
The list should be closed when clicking outside of it. When I click as a user the list closes but when Cypress clicks it does not close it.

### Test code to reproduce
Here is the repro project:

[cypress-vuetify-test.zip](https://github.com/cypress-io/cypress/files/5109469/cypress-vuetify-test.zip)

To run Vue.js app you need to execute:

```
npm install
npm run serve
```

then to launch Cypress you need to run:

```
npm run cypress:open
```

### Versions
Cypress: 5.0.0
Operating system: Windows 10
Browser: Chrome  84.0.4147.135 (but on other browsers this bug is also present)

**Comment by jennifer-shehane** [comment]
This click does not perform the correct action because the v-select has some code that prevents programmatic clicks (outside of native real clicks), which is what Cypress is doing. 

Their code looks like this:

```js
  // If click was triggered programmaticaly (domEl.click()) then
  // it shouldn't be treated as click-outside
  // Chrome/Firefox support isTrusted property
  // IE/Edge support pointerType property (empty if not triggered
  // by pointing device)
  if (('isTrusted' in e && !e.isTrusted) ||
    ('pointerType' in e && !e.pointerType)
  ) return
```

The Cypress click event will have `isTrusted` evaluate to `false`, so the code returns and does not trigger any of the actions that would normally be performed. 

<img width="502" alt="Screen Shot 2020-08-24 at 1 44 49 PM" src="https://user-images.githubusercontent.com/1271364/91014903-3f0e7a00-e610-11ea-8a71-57ae517530d4.png">

I don't know why they have such a strict check here. This [`isTrusted`](https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted) prop is readonly, so can't be set. We'd have to issue native events to fix this I believe. #311

**Comment by sainthkh** [comment]
The problem is solved at vuetifyjs/vuetify#12109. So, use the latest vuetify. As far as I checked, it worked with 2.3.19.

**Comment by NigaAlexandru** [comment]
Similar issue happening when using cy.select(), the event emitted has isTrusted prop evaluated to false, i tried triggering a "change" event but that didn't work

<img width="365" height="661" alt="Image" src="https://github.com/user-attachments/assets/896b48a4-114a-4f07-8341-25bf59cf875d" />

sample method:
const selectCountry = country => {
    cy.get('#country').select(country)
}
